test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -26,6 +26,14 @@ Describe 'Basic Job Tests' -Tags 'CI' {
 
     Context 'Basic tests' {
 
+        BeforeAll {
+            $invalidPathTestCases = @(
+                @{ path = "This is an invalid path"; case = "invalid path"}
+                @{ path = ""; case = "empty string"}
+                @{ path = " "; case = "whitespace string (single space)"}
```

Comment:
Please add expected FullyQualifiedErrorId here.

Reply:
We seem to be getting two different error ids.

`DirectoryNotFoundException,Microsoft.PowerShell.Commands.StartJobCommand` and `ParameterArgumentValidationError,Microsoft.PowerShell.Commands.StartJobCommand`

Reply:
fixed in a8b2bfb8915b815a708a4b95711ea22b989e7159